src/viur/core/modules/user.py

```diff
@@ -210,6 +210,10 @@ def toDB(cls, skel, *args, **kwargs):
                             if right in module.accessRights:
                                 access.add(f"{name}-{right}")
 
+                                # special case: "edit" and "delete" actions require "view" as well!
+                                if right in ("edit", "delete") and "view" in module.accessRights:
```

Comment:
Shouldn't this be `not in`?

```suggestion
                                if right in ("edit", "delete") and "view" not in module.accessRights:
```
or an `or`?

```suggestion
                                if right in ("edit", "delete") or "view" in module.accessRights:
```

Reply:
Nope, because "view" should only be set when it's a valid access right for the module. The condition is correct, and tested. The `access` variable in this context is a set already.